docs/diagrams/BetterModelClassDiagram.puml

```diff
@@ -16,8 +16,9 @@ UniqueStudentList -right-> Student
 
 Student -up-> "*" Tag
 
-Student *--> Name
-Student *--> Phone
-Student *--> Email
-Student *--> Address
+Student *--> "1" Name
```

Comment:
Good change since there are checks in place that ensure that these attributes are not null either